cinema/views.py

```diff
@@ -45,9 +74,32 @@ def get_serializer_class(self):
 
 
 class MovieSessionViewSet(viewsets.ModelViewSet):
-    queryset = MovieSession.objects.all()
+    queryset = (
```

Comment:
this calculation will be done no matter the "action" (list, retrieve etc) - do we need it each time?